ch5/tutorial/crud_pydantic.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+
+class CreateIn(BaseModel):
+    name: str
+    nickname: str
+
+
+class CreateOut(BaseModel):
+    status: str
+    id: int
+
+
+# Create a FastAPI instance
+app = FastAPI()
+
+# User database
+USER_DB = {}
+
+# Fail response
+NAME_NOT_FOUND = HTTPException(status_code=400, detail="Name not found.")
+
+
+@app.post("/users", response_model=CreateOut)
+def create_user(user: CreateIn):
+    USER_DB[user.name] = user.nickname
+    user_dict = user.dict()
+    user_dict["status"] = "success"
+    user_dict["id"] = len(USER_DB)
```

Comment:
```suggestion
    user_dict["id_length"] = len(USER_DB)
```
`id_length` or `id_size` 는 어떤가요? `id` 라고 하기에는 `len` 이 붙어있어서요

Reply:
무시!